cli/actions/run.go

```diff
@@ -31,24 +30,21 @@ type RunContainersOptions struct {
 }
 
 func (s *sedgeActions) RunContainers(options RunContainersOptions) error {
-	upCmd := s.commandRunner.BuildDockerComposeUpCMD(commands.DockerComposeUpOptions{
+	err := s.composeManager.Up(commands.DockerComposeUpOptions{
```

Comment:
It looks like the `DockerCompose*Options` should fall under the `compose` package rather than under the `commands` (after this refactor). WDYT

Reply:
The `ComposeManager` uses the `CommandRunner` (in `commands` pkg) interface to build the docker compose commands (which use these options). I think it would be better if we keep the `DockerCompose*Options` in the `commands` pkg to also prevent an "import cycle" issue.